disruption_generator/parsers/experiment_parser.py

```diff
@@ -89,15 +115,23 @@ def _init_actions(element_trigger):
 
         _scenarios = []
         for disrupt_action in doc:
-            name = disrupt_action[config.DISRUPT_ACTION_STR][0][config.NAME_STR]
+            name = disrupt_action[config.DISRUPT_ACTION_KEY][0][
```

Comment:
Why is there [0] index? In example.yaml, `name` key is directly under `disrupt_action`.

Reply:
Cause it bring a 1 element list